Modix.Data/Repositories/EmojiRepository.cs

```diff
@@ -277,7 +252,8 @@ limit 1
                     )
```

Comment:
Should the `"Timestamp" >= :StartTimestamp"` clause not be swapped out for `{dateClause}` for both queries?

Reply:
Good catch, will fix.